consul/data_source_consul_config_entry_v2_exported_services_test.go

```diff
@@ -0,0 +1,116 @@
+package consul
+
+import (
+	"regexp"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+)
+
+func TestAccDataExportedServicesV2_basic(t *testing.T) {
+	providers, client := startTestServer(t)
+
+	resource.Test(t, resource.TestCase{
+		Providers: providers,
+		PreCheck:  func() { skipTestOnConsulCommunityEdition(t) },
+		Steps: []resource.TestStep{
+			{
+				Config:      testAccDataSourceExportedServicesV2ConfigNotFound,
+				SkipFunc:    skipIfConsulVersionLT(client, "1.18.0"),
+				ExpectError: regexp.MustCompile(`exported services config not found: not-found`),
+			},
+			{
+				Config:   testAccDataSourceExportedServicesV2ConfigBasic,
+				SkipFunc: skipIfConsulVersionLT(client, "1.18.0"),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr("data.consul_config_entry_v2_exported_services.read", "name", "test"),
+					resource.TestCheckResourceAttr("data.consul_config_entry_v2_exported_services.read", "kind", "ExportedServices"),
+					resource.TestCheckResourceAttr("data.consul_config_entry_v2_exported_services.read", "namespace", "default"),
+					resource.TestCheckResourceAttr("data.consul_config_entry_v2_exported_services.read", "partition", "default"),
+					resource.TestCheckResourceAttr("data.consul_config_entry_v2_exported_services.read", "services.0", "s1"),
+					resource.TestCheckResourceAttr("data.consul_config_entry_v2_exported_services.read", "sameness_group_consumers.0", "sg1"),
+				),
+			},
+			{
+				Config:   testAccDataSourceNamespaceExportedServicesV2ConfigBasic,
+				SkipFunc: skipIfConsulVersionLT(client, "1.18.0"),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr("data.consul_config_entry_v2_exported_services.read", "name", "test"),
+					resource.TestCheckResourceAttr("data.consul_config_entry_v2_exported_services.read", "kind", "NamespaceExportedServices"),
+					resource.TestCheckResourceAttr("data.consul_config_entry_v2_exported_services.read", "partition", "default"),
+					resource.TestCheckResourceAttr("data.consul_config_entry_v2_exported_services.read", "partition_consumers.0", "default"),
+				),
+			},
+			{
+				Config:   testAccDataSourcePartitionExportedServicesV2ConfigBasic,
+				SkipFunc: skipIfConsulVersionLT(client, "1.18.0"),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr("data.consul_config_entry_v2_exported_services.read", "name", "test"),
+					resource.TestCheckResourceAttr("data.consul_config_entry_v2_exported_services.read", "kind", "PartitionExportedServices"),
+					resource.TestCheckResourceAttr("data.consul_config_entry_v2_exported_services.read", "partition", "default"),
+					resource.TestCheckResourceAttr("data.consul_config_entry_v2_exported_services.read", "peer_consumers.0", "peer1"),
+				),
+			},
+		},
+	})
+}
+
+const testAccDataSourcePartitionExportedServicesV2ConfigBasic = `
+resource "consul_config_entry_v2_exported_services" "test" {
+	name = "test"
```

Comment:
Is this indentation intended?

Reply:
it seems to be the convention for the other data_source_consul_*_test.go cases